caikit/config/config.yml

```diff
@@ -195,6 +195,12 @@ runtime:
         output_dir: training_output
         save_with_id: true
 
+        model_saver_plugins:
```

Comment:
We should think about whether this lives outside of `runtime` near `data_streams`